src/components/JobCard/JobList/JobList.styled.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+
+export const StyledJobsList = styled.section`
+	display: flex;
+	flex-direction: column;
+	gap: 3rem;
+	align-items: center;
+	justify-content: center;
+	width: 100%;
+
+	@media screen and (min-width: 800px) {
+		flex-direction: row;
+		flex-wrap: wrap;
+        padding: 0 5rem 0 5rem;
+	}
+
+
+	@media screen and (min-width: 1300px) {
+		flex-direction: row;
+		flex-wrap: wrap;
+        width: 90%;
+        padding: 0 8rem 0 8rem;
+	}
+`;
+
+export const StyledJobsContainer = styled.div`
+    width: 100%;
```

Comment:
Your CSS is not organize. Stick to one order of selector. For example https://9elements.com/css-rule-order?fbclid=IwAR0SpJuCZk6hQIbarYSSYQ35h33lex7BBQ851Oa04B19IiUgtkiKFpT9puI